mousestyles/path_diversity/path_features_advanced_proposed.py

```diff
@@ -0,0 +1,71 @@
+from future import print_function, absolute_import, division
+import numpy as np
+import pandas as pd
+from mousestyles.path_diversity.path_features import angle_between
```

Comment:
I originally had this line to calculate center angles in the path, but since @alannaiverson didn't pick center angles in this file, right now it is unnecessary. If we decide to include center angles in the output of this function, we need it.